src/nrnoc/cabcode.cpp

```diff
@@ -2009,12 +2008,12 @@ Node* node_ptr(Section* sec, double x, double* parea) {
 	return nd;
 }
 
-int nrn_get_mechtype(const char* mechname) {
+extern "C" int nrn_get_mechtype(const char* mechname) {
 	Symbol* s;
 	s = hoc_lookup(mechname);
 	assert(s);
 	if (s->type == TEMPLATE) {
-		s = hoc_table_lookup(mechname, s->u.template->symtable);
+		s = hoc_table_lookup(mechname, s->u.ctemplate->symtable);
 		assert(s && s->type == MECHANISM);
```

Comment:
I now see `u.ctemplate` vs `u.template` thing. Was this used in some user codes? 🤔

Reply:
Yes in the VERBATIM blocks. That's why I added `verbatim_adjust` (will document that)